cmd/device-worker/main.go

```diff
@@ -145,8 +146,24 @@ func main() {
 
 	reg := registration2.NewRegistration(&hw, deviceOs, dispatcherClient, configManager, hbs, wl, dataMonitor, metricsStore, systemMetricsWatcher)
 
+	// defined how to connect to hosts
+	ansiblePlaybookConnectionOptions := &options.AnsibleConnectionOptions{ //FIXME
```

Comment:
Is using ssh-keys to connect no? Should we add the ssh-keys for this?  or something?

Reply:
Connection local is not using ssh-keys.